pyoffers/models/constants.py

```diff
@@ -0,0 +1 @@
+APPLICATION_CONTROLLER_NAME = 'Application'
```

Comment:
Seems to be redundant, because of the comment below.

Reply:
removed